<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTextItemAddDialog</class>
 <widget class="QDialog" name="CTextItemAddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>GPS source</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ui::controls::CGpsSourceSelect" name="pwGpsSourceSelect">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbDataType">
     <property name="title">
      <string>Data type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbLongitude">
        <property name="text">
         <string>Longitude</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="rbLatitude">
        <property name="text">
         <string>Latitude</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="rbSpeedOverGround">
        <property name="text">
         <string>Speed over ground</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="rbMode">
        <property name="text">
         <string>Mode</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="rbAltitude">
        <property name="text">
         <string>Altitude</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="rbPQTotal">
        <property name="text">
         <string>Position quality total</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="rbPQHorizontal">
        <property name="text">
         <string>Position quality horizontal</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QRadioButton" name="rbPQVertical">
        <property name="text">
         <string>Position quality vertical</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="rbCourseFromNorth">
        <property name="text">
         <string>Cource from north</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QRadioButton" name="rbStatus">
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QRadioButton" name="rbSatellitesInLastFix">
        <property name="text">
         <string>Satellites in lst fix</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rbRateOfClimb">
        <property name="text">
         <string>Vertical speed</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="rbSentence">
        <property name="text">
         <string>Sentence</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbAdd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res/resources.qrc">
         <normaloff>:/icons/item-add.png</normaloff>:/icons/item-add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbClose">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res/resources.qrc">
         <normaloff>:/icons/dialog-cancel.png</normaloff>:/icons/dialog-cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ui::controls::CGpsSourceSelect</class>
   <extends>QToolButton</extends>
   <header>cgpssourceselect.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../res/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked()</signal>
   <receiver>CTextItemAddDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
